### PR TITLE
[office-js-preview] Pulling latest type definitions from CDN

### PR DESCRIPTION
**Pull request by alison-mk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
Beta URL: https://appsforoffice.microsoft.com/lib/beta/hosted/office.debug.js
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -21832,7 +21832,7 @@ declare namespace Excel {
     interface BindingSelectionChangedEventArgs {
         /**
          *
-         * Gets a temporary `Binding` object that contains the ID of the `Binding` object that raised the event. Use that ID with `BindingCollection.getItem(id)` to get the binding.
+         * Gets a temporary `Binding` object that contains the ID of the `Binding` object that raised the event. Use that ID with `BindingCollection.getItem(id)` to get the binding.
```
Comment (AlexJerabek) [inline types/office-js-preview/index.d.ts:21835]: What is actually changing here?

**Comment by alison-mk** [comment]
Closing PR for now due to internal conversation.

File: types/office-js-preview/index.d.ts (hunk 1 above)
Comment (wandyezj) [reply types/office-js-preview/index.d.ts:21835]: whitespace. These were somehow non standard spaces before.

**Review by wandyezj — APPROVED** [review]

**Comment by alison-mk** [comment]
Ready to merge

**Pull request merged**